Directory.Packages.props

```diff
@@ -12,5 +12,7 @@
     <PackageVersion Include="Moq" Version="4.14.3" />
     <PackageVersion Include="xunit" Version="2.4.0" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageVersion Include="Microsoft.Azure.WebJobs.Extensions.Sql" Version="0.1.223-preview" />
```

Comment:
do we need this here? seems weird to have this in the root

Reply:
nope not needed, removed it, thanks